unit_tests/test_coredump.py

```diff
@@ -84,10 +85,14 @@ def save_results(self, filepath):
         return Pickler.save_to_file(filepath, self._localize_results())
 
 
-class CoredumpExportTestBase(unittest.TestCase):
+class CoredumpExportTestBase(unittest.TestCase, EventsUtilsMixin):
     maxDiff = None
     test_data_folder: str = None
 
+    @classmethod
+    def setUpClass(cls):
+        cls.setup_events_processes(events_device=True, events_main_device=False, registry_patcher=True)
```

Comment:
you are missing a 
````
    @classmethod
    def tearDownClass(cls) -> None:
        cls.teardown_events_processes()
```

regardless I would advice to use the `events` fixture intend, moving to more pytest style for those things